enterprise/coderd/provisionerkeys.go

```diff
@@ -200,17 +200,44 @@ func (api *API) deleteProvisionerKey(rw http.ResponseWriter, r *http.Request) {
 	httpapi.Write(ctx, rw, http.StatusNoContent, nil)
 }
 
+// @Summary Fetch provisioner key details
+// @ID fetch-provisioner-key-details
+// @Security CoderSessionToken
```

Comment:
This comment should actually be updated as a session token here isn't sufficient auth. Sorry I didn't catch this sooner!

Reply:
I checked the definition of the `/provisionerdaemons/serve` endpoint and it looks like it also specifies `@Security CoderSessionToken`. It looks like that's the only value we allow there currently -- `swaggerparser.go` enforces this with some special cases for certain endpoints.

Reply:
As discussed , there's currently some extra logic to change in `swaggerparser.go` -> https://github.com/coder/coder/blob/main/coderd/coderdtest/swaggerparser.go#L302

I can either keep it like that and create the follow-up issue / PR quickly or add this endpoint to the ignore list for now - as you prefer.

Reply:
Let's keep it as-is for the moment and create a follow-up PR to address this.